addons/mrp/migrations/8.0.1.1/post-migrate.py

```diff
@@ -95,17 +95,36 @@ def fix_domains(cr, pool):
 
 def update_stock_moves(cr, pool):
     stock_move_obj = pool['stock.move']
-    sm_ids = stock_move_obj.search(cr, uid, [])
     mrp_production_obj = pool['mrp.production']
     location_obj = pool['stock.location']
-    location_id = location_obj.search(cr, uid, [('name', '=', 'Production')])
+
+    # TODO: improve production location search, using other criteria than name
+    location_ids = location_obj.search(cr, uid, [('name', '=', 'Production')])
```

Comment:
In fact, I just tried to improved this migration. But, yes I think that in some case (rename, ..), migration could fail. (This is the objective of my comment).
And no, I didn't find any other trick to know if a location is a production location. If you have any idea...